src/appleseed/renderer/kernel/lighting/pathvertex.h

```diff
@@ -93,6 +93,9 @@ class PathVertex
     ScatteringMode::Mode        m_aov_mode;
     Spectrum                    m_albedo;
     bool                        m_albedo_saved;
+    Spectrum                    m_sample_refraction_value;
```

Comment:
We need to be very careful about the amount of data we store in this structure since we may keep large numbers of path vertices in memory in bidirectional tracing methods. Why do we need these two?